po/zh_TW.po

```diff
@@ -18,336 +18,341 @@ msgstr ""
 
 #: data/launcher.desktop.in:3 data/metainfo.appdata.xml.in:7
 msgid "Clairvoyant"
-msgstr ""
+msgstr "Clairvoyant"
 
 #: data/launcher.desktop.in:4
 msgid "Fortune Teller"
-msgstr ""
+msgstr "算命師"
 
 #: data/launcher.desktop.in:5
 msgid "Ask questions and get psychic answers"
-msgstr ""
+msgstr "問問題，得答案"
 
 #: data/launcher.desktop.in:11
 msgid "magic;eight;ball;8;crystal;🔮;medium;🎱;"
-msgstr ""
+msgstr ";魔法;八號;球;水晶;🔮;中等;🎱;"
 
 #: data/metainfo.appdata.xml.in:8
 msgid "Ask questions, get psychic answers"
-msgstr ""
+msgstr "問問題，得答案"
 
 #: data/metainfo.appdata.xml.in:10
 msgid ""
 "Does he love you? Should you have pizza for dinner? Is there such thing as a "
 "stupid question? Discover the answers to these questions and more with "
 "Clairvoyant, the magic 8-ball inspired fortune teller."
 msgstr ""
+"他喜歡你嗎？台灣會被武統嗎？X 會換回 Twitter 標誌嗎？"
```

Comment:
Yeah I would request that you change this text, please. The original is supposed to be light-hearted and inconsequential and I don't think I want to have the text talking about specific people, companies, and countries.

Reply:
@olivertzeng could you revisit this line, please? I would like to keep it light-hearted and not mention specific people, companies, and countries.

Reply:
Solved